tests/ecmp/test_fgnhg.py

```diff
@@ -316,15 +334,15 @@ def common_setup_teardown(tbinfo, duthost):
         pytest.skip("Unsupported platform")
 
     try:
-        host_facts  = duthost.setup()['ansible_facts']
-        mg_facts   = duthost.minigraph_facts(host=duthost.hostname)['ansible_facts']
+        host_facts = duthost.setup()['ansible_facts']
+        mg_facts = duthost.minigraph_facts(host=duthost.hostname)['ansible_facts']
```

Comment:
Recommend using mg_facts defined here, across all the minigraph config generation code as an optimization.

Reply:
Addressed